packages/cookie-theme-toggler/src/styles.tsx

```diff
@@ -11,4 +11,21 @@ export const CookieThemeTogglerStyle = styled(Button).attrs({
   padding-right: 10px;
   line-height: 0;
   font-size: 0;
+
+  width: 44px;
+  height: 44px;
+`
+
+export const IconWrapper = styled.span`
+  position: absolute;
+  top: -12px;
+  left: -12px;
+`
+
+export const LightWrapper = styled(IconWrapper)`
+  visibility: var(--lido-lightModeVisibility);
```

Comment:
Isn't it possible to just define --lido-visibility and overwrite it?

Reply:
i dont understand. pls more context

Reply:
> Isn't it possible to just define --lido-visibility and overwrite it?

no, we do not have logical operators in CSS, so we cannot do like `var(--visibility) ? visible : hidden`, so we should define separate var for things that should be hidden in light theme and for dark theme